### ISSUE TITLE
Add color to NavSatFix/LocationFix messages in Map panel

### ISSUE THREAD

**Issue by kis-balazs** [foxglove/studio#2798]
Would there be a possibility to add color to a specific type of GPS message either by the type Message.sensor_msgs__NavSatFix or in any other way, when trying to use multiple GPS devices on the same map?

**Comment by jtbandes** [comment]
This is a good feature request. I think it's not specific to Node Playground, but just a feature that we should add to the Map panel. Each topic can have a separate color, like each dataset does in the Plot panel.

**Comment by kis-balazs** [comment]
Thanks, it seemed useful to me since I am using multiple GPS sensors at a time and it is very-very hard to follow which one is which.

When could we expect updates about this topic?

**Comment by kis-balazs** [comment]
Hello @jtbandes!

Could we have a short update/feedback on this idea? :)

**Comment by amacneil** [comment]
@defunctzombie thoughts on adding this to `foxglove.LocationFix` (potentially in addition to allowing topic color override, similar to marker messages)?

**Comment by defunctzombie** [comment]
@amacneil The option I'd probably start with is what we do in the plot panel - which is to round-robin different colors when we see multiple topics. I'd probably leave this up to configure in the layout rather than the message.

**Comment by defunctzombie** [comment]
@kis-balazs We've added this to the March product roadmap plan.

**Comment by kis-balazs** [comment]
@defunctzombie sounds lovely, thank you very much! Looking forward to it! :)

**Comment by amacneil** [comment]
@kis-balazs do you need the ability to customize the color per-message, or only a single color per-topic?

**Comment by kis-balazs** [comment]
I would suggest color/topic would be a first approach. More relevant, from my pov. Thanks!
